grafana-plugin/package.json

```diff
@@ -170,13 +170,14 @@
     "react-string-replace": "^0.4.4",
     "react-transition-group": "^4.4.5",
     "react-use": "^17.4.0",
-    "stylelint": "^13.13.1",
+    "stylelint": "^14.0.0",
     "stylelint-config-standard": "^22.0.0",
     "throttle-debounce": "^2.1.0",
     "tinycolor2": "^1.6.0",
     "tslib": "2.5.3"
   },
   "resolutions": {
-    "braces": "3.0.3"
+    "braces": "3.0.3",
+    "micromatch": "4.0.6"
```

Comment:
patches https://github.com/grafana/oncall-private/issues/2876 (and appeases the failing `synk test` 🙂)